jobs/README.md

```diff
@@ -4,6 +4,13 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## Front-end developer at [Ortec for Communications](https://www.orteccommunications.com)
+
+Looking for the next challenge in your career? We are hiring! We are currently seeking a full time mobile developer to join our team in Heraklion. Come and talk to us for an exciting opportunity at ORTEC For Communications.
```

Comment:
@pappaschris title says "front-end" developer, description says "mobile" developer. 

If this is on purpose, happy to merge. Just thought I'd check ;)

Reply:
don't worry. merge as is :P

Reply:
👍